templates/commands/goldentest/new_test_cli_test.go

```diff
@@ -307,3 +310,127 @@ func TestNewTestFlags_Parse(t *testing.T) {
 		})
 	}
 }
+
+func TestNewTestPrompt(t *testing.T) {
```

Comment:
I feel it is cleaner to separate the prompt test case to its own test instead of combining them with other test cases that won't need some golang channel.

Reply:
Yeah, a separate test makes sense to me, like you've done it here.